packages/nodes-base/nodes/SyncroMSP/v1/actions/customer/getAll/description.ts

```diff
@@ -0,0 +1,109 @@
+import {
+	CustomerProperties,
+} from '../../Interfaces';
+
+export const customerGetAllDescription: CustomerProperties = [
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		displayOptions: {
+			show: {
+				resource: [
+					'customer',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		default: false,
+		noDataExpression: true,
+		description: 'If all results should be returned or only up to a given limit',
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		displayOptions: {
+			show: {
+				resource: [
+					'customer',
+				],
+				operation: [
+					'getAll',
+				],
+				returnAll: [
+					false,
+				],
+			},
+		},
+		default: 25,
+		description: 'Limit the number of rows returned',
+	},
+	{
+		displayName: 'Filters',
+		name: 'additionalFilters',
+		type: 'collection',
+		placeholder: 'Add Filter',
+		displayOptions: {
+			show: {
+				resource: [
+					'customer',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		default: {},
+		options: [
+			{
+				displayName: 'Business Name',
+				name: 'businessName',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Email',
+				name: 'email',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'First Name',
+				name: 'firstname',
```

Comment:
In some locations we use "firstname" in some other "firstName". Same with lastname. That should be consistent.